src/Microsoft.Identity.Web.TokenAcquisition/PublicAPI/net6.0/InternalAPI.Shipped.txt

```diff
@@ -226,12 +226,12 @@ Microsoft.Identity.Web.MergedOptionsStore.MergedOptionsStore() -> void
 Microsoft.Identity.Web.MicrosoftIdentityAppCallsWebApiAuthenticationBuilder.MicrosoftIdentityAppCallsWebApiAuthenticationBuilder(Microsoft.Extensions.DependencyInjection.IServiceCollection! services, Microsoft.Extensions.Configuration.IConfigurationSection? configurationSection = null) -> void
 Microsoft.Identity.Web.MicrosoftIdentityApplicationOptionsMerger
 Microsoft.Identity.Web.MicrosoftIdentityApplicationOptionsMerger.MicrosoftIdentityApplicationOptionsMerger(Microsoft.Identity.Web.IMergedOptionsStore! mergedOptions) -> void
-Microsoft.Identity.Web.MicrosoftIdentityApplicationOptionsMerger.PostConfigure(string! name, Microsoft.Identity.Abstractions.MicrosoftIdentityApplicationOptions! options) -> void
+Microsoft.Identity.Web.MicrosoftIdentityApplicationOptionsMerger.PostConfigure(string? name, Microsoft.Identity.Abstractions.MicrosoftIdentityApplicationOptions! options) -> void
 Microsoft.Identity.Web.MicrosoftIdentityOptions.HasClientCredentials.get -> bool
 Microsoft.Identity.Web.MicrosoftIdentityOptions.IsB2C.get -> bool
 Microsoft.Identity.Web.MicrosoftIdentityOptionsMerger
 Microsoft.Identity.Web.MicrosoftIdentityOptionsMerger.MicrosoftIdentityOptionsMerger(Microsoft.Identity.Web.IMergedOptionsStore! mergedOptions) -> void
-Microsoft.Identity.Web.MicrosoftIdentityOptionsMerger.PostConfigure(string! name, Microsoft.Identity.Web.MicrosoftIdentityOptions! options) -> void
+Microsoft.Identity.Web.MicrosoftIdentityOptionsMerger.PostConfigure(string? name, Microsoft.Identity.Web.MicrosoftIdentityOptions! options) -> void
```

Comment:
What is causing these changes? It should be unrelated to the dependency version change.

Reply:
Use of 7.0.0 for net6.0.

Reply:
which I think you should rollback

Reply:
Reverted